site/content/entry/yuzu-progress-report-may-2023/index.md

```diff
@@ -0,0 +1,832 @@
+
+{{< imgs
+	"./VSync.png| Android has no tolerance for tearing."
+  >}}
+
+Per your writer’s ~~begging~~ request, [toastUnlimited](https://github.com/lat9nq) updated the old VSync toggle in Graphics, Advanced, for a {{< gh-hovercard "10125" "proper drop list" >}} mentioning all available Vulkan options.
+
+{{< imgs
+	"./mailbox.png| If it is an option, pick Mailbox."
+  >}}
+
+So, for example, if someone with an AMD or Intel GPU on Windows wants to play `Super Smash Bros. Ultimate` with a 120 FPS mod on a 60 Hz display, they now have the option to use Immediate mode, which is what the emulator switches to when the player unlocks the framerate with Ctrl + U.
+The rest can just enjoy Chad Mailbox.
+
+We recommend setting the driver’s control panel to Enhanced/Fast sync for the best results.
+
+{{< single-title-imgs
+    "Press F to sync"
+    "./amd.png"
+    "./nvidia.png"
+    "./intel.png"
+    >}}
+
+Maide found an issue related to the size of the pipeline cache.
+Yuzu used to return the size of the shader code in bytes, which was later used to resize the unsigned 64 bit integer array in charge of storing the cache, which made the array 8 times larger than it was supposed to be.
+This is fixed by {{< gh-hovercard "10145" "using the length of the array" >}} instead of the size in bytes.
+
+A common complaint from 8GB of RAM users is that games become unstable over time. This isn't necessarily a memory leak — it's likely just the system running out of RAM as new shaders are added to the pipeline cache.
+This change greatly benefits them.
+~~But it’s 2023, you should seriously consider getting 16GB by now. Tears of the Kingdom is not Super Mario Odyssey.~~
+
+ASTC continues to be in the news here — this time a problem with {{< gh-hovercard "10206" "3D ASTC" >}} textures, because it isn’t enough to deal with flat 2D ones.
+A bug in their implementation caused the level Frantic Fields in `Donkey Kong Country: Tropical Freeze` to render incorrectly.
+Some tweaks by the ‘hawk, and the Kong army is back in action.
+
+{{< imgs
+	"./dk.png| Quicksands no more! (Donkey Kong Country: Tropical Freeze)"
+  >}}
+
+The buffer cache, responsible for storing information for the GPU to process at the right time.
+The buffer can be modified by either the GPU or the CPU, so yuzu would set the buffer to either one, then wait for the other type, when in reality, {{< gh-hovercard "10216" "checking for one state" >}} before setting to the other would make much more sense.
+Maide noticed this discrepancy and set out to correct it, streamlining the code.
+
+Another mishap in the buffer cache affected rendering in `Dokapon Kingdom Connect`.
+
+{{< single-title-imgs
+    "There’s still some room for improvement! (Dokapon Kingdom Connect)"
+    "./dkgbug.mp4"
+    "./dkgfix.mp4"
+    >}}
+
+Since the old code would track buffers globally, some draws would end up ignoring channel swaps, leading to bad values in the uniform buffer.
```

Comment:
```suggestion
Since the old code would track buffers globally, some draws would end up ignoring channel swaps, leading to leftover values from the wrong channel getting bound in uniform buffers.
```